JellyfishClient/src/main/java/com/jellyfishdev/jellyfishclient/JellyfishClientListener.kt

```diff
@@ -38,44 +38,36 @@ interface JellyfishClientListener {
     fun onAuthError()
 
     /**
-     * Emitted when the connection is closed
-     */
-    fun onDisconnected() {
-        Timber.i("Client disconnected")
-    }
-
-    /**
-     * Called when peer was accepted.
+     * Emitted when local user is connected to jellyfish.
      */
-    fun onJoinSuccess(peerID: String, peersInRoom: List<Peer>)
+    fun onConnected(peerID: String, peersInRoom: List<Endpoint>)
 
     /**
-     * Called when peer was not accepted
-     * @param metadata - Pass thru for client application to communicate further actions to frontend
+     * Emitted when there was an error while connecting to the jellyfish.
      */
-    fun onJoinError(metadata: Any)
+    fun onConnectError(metadata: Any)
 
     /**
-     * Called every time a local peer is removed by the server.
+     * Emitted when the connection is closed
      */
-    fun onRemoved(reason: String)
+    fun onDisconnected() {
+        Timber.i("Client disconnected")
+    }
 
     /**
      * Called each time new peer joins the room.
      */
-    fun onPeerJoined(peer: Peer)
+    fun onPeerJoined(peer: Endpoint)
```

Comment:
this `Endpoint` here doesn't look good, you can try using `typealias` in kotlin so that the user would see Peer instead of Endpoint

the ts sdk has something similar https://github.com/jellyfish-dev/ts-client-sdk/blob/main/src/JellyfishClient.ts#L15

Reply:
Done.